pkg/pillar/cmd/zedmanager/zedmanager.go

```diff
@@ -539,6 +567,38 @@ func checkDelayedStartApps(ctx *zedmanagerContext) {
 	}
 }
 
+func handleAA(ctx *zedmanagerContext, status types.AssignableAdapters, key string) {
+	log.Functionf("handleAA(%s)", status.Key())
+	if key != "global" {
+		log.Functionf("handleAAImpl: ignoring %s", key)
```

Comment:
Some instances of "Impl" are still in log messages.

Reply:
Fixed